tests/model_serving/model_server/llmd/utils.py

```
@@ -285,6 +351,70 @@ def _curl_post(
+        llmisvc: The LLMInferenceService to query.
+        token: Optional bearer token for authentication.
+        insecure: Skip TLS verification (default True).
+
+    Returns:
+        The vLLM version string (e.g. "0.8.5").
+
+    Raises:
+        ValueError: If the response cannot be parsed or the endpoint returns an error.
+    """
+    base_url = (
+        _get_disconnected_inference_url(llmisvc)
```

Flake8 (7.3.0)

[error] undefined name '_get_disconnected_inference_url'

(F821)